src/rust-crypto/RoomEncryptor.ts

```diff
@@ -46,8 +46,11 @@ export class RoomEncryptor {
     /** whether the room members have been loaded and tracked for the first time */
     private lazyLoadedMembersResolved = false;
 
-    /** Ensures that there is only one call to shareRoomKeys at a time */
-    private currentShareRoomKeyPromise = Promise.resolve();
+    /**
+     * Ensures that there is only one encryption operation at a time for that room
+     * An encryption operation is either a prepareForEncryption or an encryptEvent call.
```

Comment:
Nits:

```suggestion
     * Ensures that there is only one encryption operation at a time for that room.
     *
     * An encryption operation is either a  {@link #prepareForEncryption} or an {@link #encryptEvent} call.
```

As well as improving the formatting, `@link` ensures that if the methods ever get renamed, we don't get left with a dangling reference (because the linter will catch it).